### PR TITLE
GitHub based challenges MVP

### PR DESCRIPTION
**Pull request by KhalidRmb**
image
Related PR on EvalAI is -- https://github.com/Cloud-CV/EvalAI/pull/2965

### CODE REVIEW

File: github/challenge_processing_script.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+
+if __name__ == "__main__":
+
+	configs = load_host_configs(HOST_CONFIG_FILE_PATH)
+	if configs:
+		HOST_AUTH_TOKEN = configs[0]
+		CHALLENGE_HOST_TEAM_PK = configs[1]
+		EVALAI_HOST_URL = configs[2]
+	else:
```
Comment (RishabhJain2018) [inline github/challenge_processing_script.py:41]: Can we please add a print message here before exiting stating what might be the issue, something like HOSY_AUTH_TOKEN or CHALLENGE_HOST_TEAM_PK or EVALAI_HOST_URL is missing.

File: github/challenge_processing_script.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	except requests.exceptions.HTTPError as err:
+		if response.status_code in EVALAI_ERROR_CODES:
+			is_token_valid = validate_token(response.json())
+			if is_token_valid:
+				error = response.json()["error"]
+				error_message = "\nFollowing errors occurred while validating the challenge config:\n{}".format(error)
+				print(error_message)
+				os.environ["CHALLENGE_ERRORS"] = error_message
+		else:
+			print("\nThere was an error: {}".format(err))
```
Comment (RishabhJain2018) [inline github/challenge_processing_script.py:75]: Please add a specific error message here -- stating `There was an error in ...`

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: github/challenge_processing_script.py (hunk 1 above)
Comment (KhalidRmb) [reply github/challenge_processing_script.py:41]: It's done inside the util method which tries to load this data. Not needed here.

**Review by RishabhJain2018 — APPROVED** [review]
LGTM! Looking forward to see it in action?

**Pull request merged**